commit 0a9fd2a9875bd60afc0f8f9316e9ce63a1eb4e87
Author: Pavel Machek <pavel@ucw.cz>
Date:   Wed Aug 1 22:28:21 2018 +0200

    ARM: dts: omap4-droid4: fix vibrations on Droid 4
    
    [ Upstream commit f4efa74c09a7eddcc12cd13208f78743763f6e7a ]
    
    Vibration GPIOs don't have anything to do with wakeup. Move it to
    normal section; this fixes vibrations on Droid 4.
    
    Fixes: a5effd968301 ("ARM: dts: omap4-droid4: Add vibrator")
    Signed-off-by: Pavel Machek <pavel@ucw.cz>
    Reviewed-by: Sebastian Reichel <sebastian.reichel@collabora.co.uk>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/arch/arm/boot/dts/omap4-droid4-xt894.dts b/arch/arm/boot/dts/omap4-droid4-xt894.dts
index e7c3c563ff8f..edc97f89fae4 100644
--- a/arch/arm/boot/dts/omap4-droid4-xt894.dts
+++ b/arch/arm/boot/dts/omap4-droid4-xt894.dts
@@ -618,15 +618,6 @@
 		OMAP4_IOPAD(0x10c, PIN_INPUT | MUX_MODE1)	/* abe_mcbsp3_fsx */
 		>;
 	};
-};
-
-&omap4_pmx_wkup {
-	usb_gpio_mux_sel2: pinmux_usb_gpio_mux_sel2_pins {
-		/* gpio_wk0 */
-		pinctrl-single,pins = <
-		OMAP4_IOPAD(0x040, PIN_OUTPUT_PULLDOWN | MUX_MODE3)
-		>;
-	};
 
 	vibrator_direction_pin: pinmux_vibrator_direction_pin {
 		pinctrl-single,pins = <
@@ -641,6 +632,15 @@
 	};
 };
 
+&omap4_pmx_wkup {
+	usb_gpio_mux_sel2: pinmux_usb_gpio_mux_sel2_pins {
+		/* gpio_wk0 */
+		pinctrl-single,pins = <
+		OMAP4_IOPAD(0x040, PIN_OUTPUT_PULLDOWN | MUX_MODE3)
+		>;
+	};
+};
+
 /*
  * As uart1 is wired to mdm6600 with rts and cts, we can use the cts pin for
  * uart1 wakeirq.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1269_0a9fd2a9875bd60afc0f8f9316e9ce63a1eb4e87.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1269_0a9fd2a9875bd60afc0f8f9316e9ce63a1eb4e87.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
